<launch>
    <arg name="config" default="data_params"/>
    <node pkg="iplanner_node" type="data_collect_node.py" name="data_collect_node" output="screen">
        <rosparam command="load" file="$(find iplanner_node)/config/$(arg config).yaml"/>
    </node>
    <!-- <node type="rviz" name="boundary_rviz" pkg="rviz" args="-d $(find data_collector_sim)/rviz/default.rviz" respawn="true"/> -->
</launch>
